docs/content/doc/help/faq.en-us.md

```diff
@@ -120,10 +120,6 @@ In Gitea, an "active" user refers to a user that has activated their account via
 
 A "login prohibited" user is a user that is not allowed to log in to Gitea anymore
 
-## Setting up logging
```

Comment:
We still need to link to this

Reply:
reverted

Reply:
I removed it before because there is already a clear "Logging configuration" in the left menu, so IMO it's not necessary to put a single link.  Keeping it is also fine.